feature-pack/pom.xml

```diff
@@ -1301,7 +1307,7 @@
 
         <dependency>
             <groupId>org.jboss.resteasy</groupId>
-            <artifactId>resteasy-validator-provider-11</artifactId>
```

Comment:
resteasy-validator-provider-11:3.7.0 artifact doesn't exist anymore? It looks like a backward compatibility blocker (cc @asoldano @msve)

Reply:
Hmmm. I replaced RESTEASY_VALIDATOR_11 in wildfly-jaxrs with RESTEASY_VALIDATOR. Maybe I should have left it and added RESTEASY_VALIDATOR.

Reply:
I restored RESTEASY_VALIDATOR_11.

Reply:
+1. If you'd just left RESTEASY_VALIDATOR_11 my guess is Galleon would see it's an alias module and would provision the target. But my guess is only having RESTEASY_VALIDATOR, Galleon would not know the alias module exists and wouldn't provision it. @jfdenise am I right?

Either way having both constants in there is the most robust.

Reply:
@bstansberry , galleon needs both, there is no support for alias. That is something that would need to be handled by galleon. Will investigate this case.